tools/ci_build/github/azure-pipelines/vienna-linux-ci-pipeline.yml

```diff
@@ -0,0 +1,19 @@
+jobs:
```

Comment:
can you rename this file such that it contains onnxruntime-server?

Reply:
Will rename it to `linux-ort-srv-ci-pipeline.yml`

Reply:
Done.